CHANGELOG.md

```diff
@@ -1,215 +0,0 @@
-# PD Change Log
-
-## v3.0.0-beta.1
```

Comment:
I think we still need to maintain changelog

Reply:
Will we resume updating them? Or guide readers to the release list for information?

Reply:
Let's guide to release list.